bootstrap/services/storm-bundle.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "STORM",
+  "registerClass": "com.hortonworks.streamline.streams.cluster.register.impl.StormServiceRegisterer",
+  "serviceUISpecification": {
+    "fields": [
+      {
+        "uiName": "UI server hostname",
+        "fieldName": "uiServerHostname",
+        "isOptional": false,
+        "tooltip": "Hostname of Storm UI server",
+        "type": "string"
+      },
+      {
+        "uiName": "Nimbuses hostnames",
+        "fieldName": "nimbusesHostnames",
+        "isOptional": false,
+        "tooltip": "Hostnames of Nimbuses",
+        "type": "array.string"
+      },
+      {
+        "uiName": "storm.yaml",
+        "fieldName": "storm.yaml",
+        "isOptional": false,
```

Comment:
@priyank5485 
Port informations are extracted from the config file. 
Maybe we can just let user input ports too, but given that we should support secured Storm cluster as well, I'd rather leave this until we define all mandatory and optional fields.